### PR TITLE
Typos and language in README

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,18 +73,18 @@ Compiling
 ---------
 
 Compiling FreeCAD requires installation of several libraries and their 
-development files such as OpenCASCADe, Coin and Qt, listed in the 
-pages below. Once this is done, FreeCAD can be simply compiled with 
-cMake. On Windows, these libraries are bundled and offered by the 
+development files such as OCCT, Coin and Qt, listed in the
```
Comment (comradekingu) [inline README.md:76]: As per https://www.opencascade.com/

File: README.md [hunk 2]
```diff
@@ -73,18 +73,18 @@ Compiling
 ---------
 
 Compiling FreeCAD requires installation of several libraries and their 
-development files such as OpenCASCADe, Coin and Qt, listed in the 
-pages below. Once this is done, FreeCAD can be simply compiled with 
-cMake. On Windows, these libraries are bundled and offered by the 
+development files such as OCCT, Coin and Qt, listed in the 
+pages below. Once this is done, FreeCAD can be compiled with 
+CMake. On Windows, these libraries are bundled and offered by the
```
Comment (comradekingu) [inline README.md:78]: As per https://cmake.org/

File: README.md [hunk 3]
```diff
@@ -25,31 +25,31 @@ Overview
...
 
 * **Designed for your needs** FreeCAD is designed to fit a wide range of uses
  including product design, mechanical engineering and architecture. 
- Whether you are a hobbyist, a programmer, an experienced CAD user, 
- a student or a teacher, you will feel right at home with FreeCAD.
+ Whether you are a hobbyist, programmer, experienced CAD user, student or teacher,
+ you will feel right at home.
 
-* **Cross platform** FreeCAD runs on Windows, Mac and Linux
+* **Cross platform** FreeCAD runs on Windows, macOS and Linux
```
Comment (comradekingu) [inline README.md:40]: As per https://en.wikipedia.org/wiki/MacOS

File: README.md [hunk 4]
```diff
@@ -25,31 +25,31 @@ Overview
 * **Freedom to build what you want**  FreeCAD is an open-source parametric 3D 
 modeler made primarily to design real-life objects of any size. 
 Parametric modeling allows you to easily modify your design by going back into 
-your model history and changing its parameters. 
+your model history to change its parameters. 
 
-* **Create 3D from 2D & back** FreeCAD allows you to sketch geometry constrained
+* **Create 3D from 2D & back** FreeCAD lets you to sketch geometry constrained
```
Comment (comradekingu) [inline README.md:30]:
```suggestion
* **Create 3D from 2D and back** FreeCAD lets you to sketch geometry constrained
```

File: README.md [hunk 5]
```diff
@@ -25,31 +25,31 @@ Overview
...
  It contains many components to adjust dimensions or extract design details from 
- 3D models to create high quality production ready drawings.
+ 3D models to create high quality production-ready drawings.
 
 * **Designed for your needs** FreeCAD is designed to fit a wide range of uses
  including product design, mechanical engineering and architecture. 
- Whether you are a hobbyist, a programmer, an experienced CAD user, 
- a student or a teacher, you will feel right at home with FreeCAD.
+ Whether you are a hobbyist, programmer, experienced CAD user, student or teacher,
+ you will feel right at home.
```
Comment (luzpaz) [inline README.md:38]: Thanks for the edit. LGTM. But there is something nagging at me. 

> you will feel right at home

Frankly, this is presumptuous on our part. Maybe something like 'one can find benefit' or something similar. Make it more neutral

File: README.md (hunk 1 above)
Comment (luzpaz) [reply README.md:76]: Maybe add (OpenCascade) in parenthesis?

> OCCT (OpenCascade)

File: README.md (hunk 5 above)
Comment (comradekingu) [inline README.md:38]:
```suggestion
 you will hopefully feel right at home.
```

File: README.md (hunk 1 above)
Comment (comradekingu) [inline README.md:76]:
```suggestion
development files such as OCCT (Open Cascade), Coin and Qt, listed in the 
```

File: README.md [hunk 6]
```diff
@@ -27,15 +27,15 @@ modeler made primarily to design real-life objects of any size.
...
+* **Create 3D from 2D and back** FreeCAD lets you to sketch geometry constrained
  2D shapes and use them as a base to build other objects. 
  It contains many components to adjust dimensions or extract design details from 
  3D models to create high quality production-ready drawings.
 
 * **Designed for your needs** FreeCAD is designed to fit a wide range of uses
  including product design, mechanical engineering and architecture. 
  Whether you are a hobbyist, programmer, experienced CAD user, student or teacher,
- you will feel right at home.
+ you will hopefully feel right at home.
```
Comment (luzpaz) [inline README.md:38]: This sentence still rubs me the wrong way, lol.  
I think we can remove line 38 altogether, actually

Comment (comradekingu) [reply README.md]: Gone :)

File: README.md [hunk 7]
```diff
@@ -33,9 +33,8 @@ your model history to change its parameters.
  3D models to create high quality production-ready drawings.
 
 * **Designed for your needs** FreeCAD is designed to fit a wide range of uses
- including product design, mechanical engineering and architecture. 
- Whether you are a hobbyist, programmer, experienced CAD user, student or teacher,
- you will hopefully feel right at home.
+including product design, mechanical engineering and architecture,
+whether you are a hobbyist, programmer, experienced CAD user, student or teacher,
```
Comment (luzpaz) [inline README.md:37]: We're almost there, lol (thx for your patience) but you need to swap the comma for a period after `teacher`  
regex: `s/teacher,$/teacher\./`

Edit: I just sent a PR to your branch :smile:

**Review by luzpaz — CHANGES_REQUESTED** [review]

File: README.md (hunk 7 above)
Comment (comradekingu) [reply README.md]: Just when I had surrendered to my faith of being here forever. :)

**Comment by wwmayer** [comment]
I hope the changes are all OK now. @luzpaz If not, feel free to create a new PR to fix open issues.

**Pull request merged**